cdap-cli/src/main/java/co/cask/cdap/cli/command/security/AddRoleToPrincipalCommand.java

```diff
@@ -47,13 +47,12 @@ public void perform(Arguments arguments, PrintStream output) throws Exception {
     String principalName = arguments.get("principal-name");
     client.addRoleToPrincipal(new Role(roleName), new Principal(principalName, Principal.PrincipalType.valueOf
       (principalType.toUpperCase())));
-    output.printf("Successfully added role '%s' to %s %s\n", roleName, principalType, principalName);
+    output.printf("Successfully added role '%s' to '%s' '%s'\n", roleName, principalType, principalName);
   }
 
   @Override
   public String getPattern() {
-    return String.format("add role <%s> to %s <%s> %s <%s>", ArgumentName.ROLE_NAME,
-                         ArgumentName.PRINCIPAL_TYPE, ArgumentName.PRINCIPAL_TYPE, ArgumentName.PRINCIPAL_NAME,
+    return String.format("add role <%s> to <%s> <%s>", ArgumentName.ROLE_NAME, ArgumentName.PRINCIPAL_TYPE,
```

Comment:
I don't think the CLI should check that. I thought about handling it in Handler but then I think we should not and keep it flexible. It's responsibility of the authorization backend to handle it. For example in some authorization system it might be possible to add a role to another. 
I am adding these checks in InMemoryAuthorizer and unit tests for them. Let me know your thoughts.
SentryAuthorizer will handler it too.